### PR TITLE
Added HP CMSL 1.6.3.1

### PR DESCRIPTION
**Pull request by sootysec**
Signed-off-by: Daniel Sutton <daniel@ducksecops.uk>

- [x ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x ] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/29351)

### CODE REVIEW

File: manifests/h/HP/HPCMSL/1.6.3.15069/HP.HPCMSL.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.1.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: HP.HPCMSL
+PackageVersion: 1.6.3.15069
```
Comment (ItzLevvie) [inline manifests/h/HP/HPCMSL/1.6.3.15069/HP.HPCMSL.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.6.3
```

File: manifests/h/HP/HPCMSL/1.6.3.15069/HP.HPCMSL.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.1.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: HP.HPCMSL
+PackageVersion: 1.6.3.15069
```
Comment (ItzLevvie) [inline manifests/h/HP/HPCMSL/1.6.3.15069/HP.HPCMSL.yaml:5]:
```suggestion
PackageVersion: 1.6.3
```

File: manifests/h/HP/HPCMSL/1.6.3.15069/HP.HPCMSL.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://hpia.hpcloud.hp.com/downloads/cmsl/hp-cmsl-1.6.3.exe
+  InstallerSha256: 914A24AE1502D5A96D53371741B412B8CA927116716AA7BBE4B3E4554ABE6F38
+  ProductCode: '{5A1AECCB-E0CE-4D2C-833C-29CCEA959448}'
```
Comment (ItzLevvie) [inline manifests/h/HP/HPCMSL/1.6.3.15069/HP.HPCMSL.installer.yaml:15]:
```suggestion
  ProductCode: '{5A1AECCB-E0CE-4D2C-833C-29CCEA959448}_is1'
```

File: manifests/h/HP/HPCMSL/1.6.3.15069/HP.HPCMSL.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+InstallerSwitches:
+    Silent: /VERYSILENT
+    SilentWithProgress: /VERYSILENT
```
Comment (ItzLevvie) [inline manifests/h/HP/HPCMSL/1.6.3.15069/HP.HPCMSL.installer.yaml:18]:
```suggestion
```

File: manifests/h/HP/HPCMSL/1.6.3.15069/HP.HPCMSL.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+PackageLocale: en-US
+Publisher: HP Development Company, L.P.
+PackageName: HP Client Management Script Library
+License: Copyright 2018-2021 HP Development Company, L.P.
+ShortDescription: HP Client Management Script Library Installer
```
Comment (ItzLevvie) [inline manifests/h/HP/HPCMSL/1.6.3.15069/HP.HPCMSL.locale.en-US.yaml:10]:
```suggestion
ShortDescription: HP Client Management Script Library is a collection of powershell functions for managing HP BIOS, and streamlining the download and organization of Softpaq downloads.
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You'll have to update the version folder as well to match `PackageVersion`.

**Review by ItzLevvie — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**